### PR TITLE
Add license FAQ question

### PR DESCRIPTION
**Pull request by ericphanson**
Based on https://github.com/JuliaRegistries/General/pull/31549#issuecomment-796671872 and https://github.com/JuliaRegistries/General/pull/31549#issuecomment-804196208, which I also link to in the answer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -212,6 +212,12 @@ Report it to the package repository.
 You can't. Package registrations are **permanent**. A version can not be overwritten in the
 registry, and code cannot be deleted.
 
+#### Can my package be registered without an [OSI approved license](https://opensource.org/licenses)?
+
+No, sorry. The registry is maintained by volunteers, and we don't have a legal team that can thoroughly review licenses. It is very easy accidentally wander into legally murky territory when combining common OSI licenses[^1] like GPL with non-OSI licenses and we don't want to subject Julia users to that hazard when installing packages registered in General. See [these](https://github.com/JuliaRegistries/General/pull/31549#issuecomment-796671872) [comments](https://github.com/JuliaRegistries/General/pull/31549#issuecomment-804196208) for more discussion.
```
Comment (fredrikekre) [inline README.md:217]:
```suggestion
No, sorry. The registry is maintained by volunteers, and we don't have a legal team that can thoroughly review licenses. It is very easy accidentally wander into legally murky territory when combining common OSI licenses[^1] like GPL with non-OSI licenses and we don't want to subject Julia users to that risk when installing packages registered in General. See [these](https://github.com/JuliaRegistries/General/pull/31549#issuecomment-796671872) [comments](https://github.com/JuliaRegistries/General/pull/31549#issuecomment-804196208) for more discussion.
```
Perhaps also use linebreaks like in the rest of the file

Comment (ericphanson) [reply README.md]: Oops, thanks

File: README.md [hunk 2]
```diff
@@ -212,6 +212,12 @@ Report it to the package repository.
 You can't. Package registrations are **permanent**. A version can not be overwritten in the
 registry, and code cannot be deleted.
 
+#### Can my package be registered without an [OSI approved license](https://opensource.org/licenses)?
+
+No, sorry. The registry is maintained by volunteers, and we don't have a legal team that can thoroughly review licenses. It is very easy accidentally wander into legally murky territory when combining common OSI licenses[^1] like GPL with non-OSI licenses and we don't want to subject Julia users to that risk when installing packages registered in General. See [these](https://github.com/JuliaRegistries/General/pull/31549#issuecomment-796671872) [comments](https://github.com/JuliaRegistries/General/pull/31549#issuecomment-804196208) for more discussion.
```
Comment (visr) [inline README.md:217]: easy accidentally -> easy to accidentally

Comment (ericphanson) [reply README.md:217]: Thanks!

```suggestion
No, sorry. The registry is maintained by volunteers, and we don't have a legal team that can thoroughly review licenses. It is very easy to accidentally wander into legally murky territory when combining common OSI licenses[^1] like GPL with non-OSI licenses and we don't want to subject Julia users to that risk when installing packages registered in General. See [these](https://github.com/JuliaRegistries/General/pull/31549#issuecomment-796671872) [comments](https://github.com/JuliaRegistries/General/pull/31549#issuecomment-804196208) for more discussion.
```

File: README.md [hunk 3]
```diff
@@ -212,6 +212,12 @@ Report it to the package repository.
 You can't. Package registrations are **permanent**. A version can not be overwritten in the
 registry, and code cannot be deleted.
 
+#### Can my package be registered without an [OSI approved license](https://opensource.org/licenses)?
+
+No, sorry. The registry is maintained by volunteers, and we don't have a legal team that can thoroughly review licenses. It is very easy to accidentally wander into legally murky territory when combining common OSI licenses[^1] like GPL with non-OSI licenses and we don't want to subject Julia users to that risk when installing packages registered in General. See [these](https://github.com/JuliaRegistries/General/pull/31549#issuecomment-796671872) [comments](https://github.com/JuliaRegistries/General/pull/31549#issuecomment-804196208) for more discussion.
+
+[^1]: Note that even within the world of OSI licenses, there are combinations of OSI licenses which are not legal to use together, such as GPL2 with Apache2.
```
Comment (ericphanson) [inline README.md:219]: Linebreaks
```suggestion
No, sorry. The registry is maintained by volunteers, and we don't have a legal team that can thoroughly review licenses.
It is very easy to accidentally wander into legally murky territory when combining common OSI licenses[^1] like GPL
with non-OSI licenses and we don't want to subject Julia users to that risk when installing packages registered in General.
See [these](https://github.com/JuliaRegistries/General/pull/31549#issuecomment-796671872) [comments]
(https://github.com/JuliaRegistries/General/pull/31549#issuecomment-804196208) for more discussion.

[^1]: Note that even within the world of OSI licenses, there are combinations of OSI licenses which are not
legal to use together, such as GPL2 with Apache2.
```

File: README.md [hunk 4]
```diff
@@ -212,6 +212,17 @@ Report it to the package repository.
 You can't. Package registrations are **permanent**. A version can not be overwritten in the
 registry, and code cannot be deleted.
 
+#### Can my package be registered without an [OSI approved license](https://opensource.org/licenses)?
+
+No, sorry. The registry is maintained by volunteers, and we don't have a legal team that can thoroughly review licenses.
```
Comment (ericphanson) [inline README.md:217]:
```suggestion
No, sorry. The registry is maintained by volunteers, and we don't have a legal team who can thoroughly review licenses.
```

**Comment by DilumAluthge** [comment]
Since this PR does have a few sentences that talk about e.g. the legality of combining different licenses, can you add a sentence at the very end that says something of the form, "we are not lawyers, and this is not legal advice"?

File: README.md [hunk 5]
```diff
@@ -212,6 +212,17 @@ Report it to the package repository.
 You can't. Package registrations are **permanent**. A version can not be overwritten in the
 registry, and code cannot be deleted.
 
+#### Can my package be registered without an [OSI approved license](https://opensource.org/licenses)?
+
+No, sorry. The registry is maintained by volunteers, and we don't have a legal team who can thoroughly review licenses.
+It is very easy to accidentally wander into legally murky territory when combining common OSI licenses[^1] like GPL
+with non-OSI licenses and we don't want to subject Julia users to that risk when installing packages registered in General.
+See [these](https://github.com/JuliaRegistries/General/pull/31549#issuecomment-796671872) [comments]
+(https://github.com/JuliaRegistries/General/pull/31549#issuecomment-804196208) for more discussion.
```
Comment (ericphanson) [inline README.md:221]:
```suggestion
(https://github.com/JuliaRegistries/General/pull/31549#issuecomment-804196208) for more discussion. We are not lawyers and this is not legal advice.
```

**Comment by ericphanson** [comment]
Ok, added

**Review by DilumAluthge — APPROVED** [review]

**Pull request merged**